src/services/io/podio/JEventSourcePODIO.cc

```diff
@@ -132,15 +132,15 @@ void JEventSourcePODIO::Open() {
     try {
 
         // Verify file exists
-        if( ! std::filesystem::exists(GetResourceName()) ){
-            // Here we go against the standard practice of throwing an error and print
-            // the message and exit immediately. This is because we want the last message
-            // on the screen to be that the file doesn't exist.
-            auto mess = fmt::format(fmt::emphasis::bold | fg(fmt::color::red),"ERROR: ");
-            mess += fmt::format(fmt::emphasis::bold, "file: {} does not exist!",  GetResourceName());
-            std::cerr << std::endl << std::endl << mess << std::endl << std::endl;
-            std::_Exit(EXIT_FAILURE);
-        }
+        // if( ! std::filesystem::exists(GetResourceName()) ){
+        //     // Here we go against the standard practice of throwing an error and print
+        //     // the message and exit immediately. This is because we want the last message
+        //     // on the screen to be that the file doesn't exist.
+        //     auto mess = fmt::format(fmt::emphasis::bold | fg(fmt::color::red),"ERROR: ");
+        //     mess += fmt::format(fmt::emphasis::bold, "file: {} does not exist!",  GetResourceName());
+        //     std::cerr << std::endl << std::endl << mess << std::endl << std::endl;
+        //     std::_Exit(EXIT_FAILURE);
+        // }
```

Comment:
Should not be a problem to remove this. How bad is the error message?

If the new message is unintelligible, we need to keep the check. Perhaps logic like in https://github.com/AIDASoft/DD4hep/pull/1283/files#diff-c3daa04c0212b4cc1ba7523f1952d41583fe46dd73aee5ec8c052061c6bbbc7fR570 using a regex like `^[A-Za-z][A-Za-z0-9+-.]*:` (based on my understanding of section 3.1 of [RFC 3986](https://www.rfc-editor.org/rfc/rfc3986)) should be enough.

Reply:
The error without this is just the error thrown by the CheckOpenable function as it already identifies that the file doesn't exist and stops.